Atypon Journals.js

```diff
@@ -226,8 +226,11 @@ function scrape(doc, url, extras) {
 
 				item.url = url;
 				item.notes = [];
+				if (tagentry){
+				var tags = tagentry.split(/\s*,\s*/)
```

Comment:
- Semikolon am Zeilenende fehlt
- Bitte auch die Einrückung aller Anweisungen innerhalb von if (tagentry) {...} korrigieren, inkl. FOR-Schleife
- Da es sich nur um lokale und nicht um globale Variablen handelt sollten diese nicht mit "var" sondern nur mit "let" deklariert werden (sowohl "tags" als auch "i")